packages/application-shell/src/components/quick-access/butler/butler.mod.css

```diff
@@ -77,6 +77,10 @@
   padding: var(--spacing-16) var(--spacing-16) var(--spacing-8) var(--spacing-8);
 }
 
+.searchText::placeholder {
```

Comment:
Question, do we have auto prefixing happening?

Reply:
I do assume so as the rollup config specifically has it: 

```js
postcssPresetEnv({
            browsers: browserslist.production,
            autoprefixer: { grid: true },
          }),
```

Reply:
Because if not we need at least the edge prefixed version  https://caniuse.com/#feat=css-placeholder

Reply:
🎉